.circleci/config.yml

```diff
@@ -64,7 +51,6 @@ jobs:
       - checkout
       - *install_sys_deps
       - *install_gitlfs
-      - *restore_cache
```

Comment:
I'd keep the cache when testing all the models.

Reply:
Yes ofcourse
Here - https://github.com/kipoi/models/commit/aaf51c17eabb07f703dc2f29a9a507550301e0b4